client/src/components/PageLayout/Header.tsx

```diff
@@ -64,85 +73,86 @@ export const Header: React.FC = () => {
           {loadingUser ? (
             <Spinner color="white" size="xl" />
           ) : (
-            <Menu>
-              <MenuButton
-                as={Button}
-                data-cy="menu-button"
-                aria-label="Options"
-                variant="outline"
-                background="gray.10"
-                px={[2, 4]}
-                py={[1, 2]}
-              >
-                Menu
-              </MenuButton>
-              <MenuList
-                paddingBlock={0}
-                display="flex"
-                flexDirection="column"
-                fontWeight="600"
-                borderRadius="5px"
-              >
-                <NextLink passHref href="/chapters">
-                  <MenuItem as="a">Chapters</MenuItem>
-                </NextLink>
+            <>
+              {!user && (
+                <Button
+                  data-cy="login-button"
+                  background="gray.10"
+                  onClick={login}
+                  fontWeight="600"
+                  width="4.5em"
+                >
+                  Login
+                </Button>
+              )}
+              <Menu>
+                <MenuButton
+                  as={Button}
+                  data-cy="menu-button"
+                  padding="0"
+                  width="4.5em"
+                  {...(user ? menuButtonStyles.login : menuButtonStyles.logout)}
+                >
+                  {user ? (
+                    <HStack spacing="0">
+                      <Avatar user={user} cursor="pointer" aria-label="menu" />
+                      <ArrowDownIcon
```

Comment:
```suggestion
                      <ChevronDownIcon
```
Ideally this should flip from down to up as you go from collapsed to expanded, but it's fine to be static for this PR.